index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <link rel="stylesheet" type="text/css" href="style/style.css">
+</head>
+
+<body>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3">
+      <header>
+        <h2>Mission: Save <span> 2 </span> pigs </h2>
+        <button id="button" type="button" class="btn btn-info btn-lg btn-block">Start Game</button>
+        <h3>Current Score: <span> 0 </span></h3>
+      </header>
+
+      <div id="box">
+
+        <div id="fireArea"> Home
+          <div id="player"></div>
+          <!-- insert bullet here -->
+
+        </div> <!-- end of fireArea-->
+
+        <div class="enemy"> </div>
+          <!-- insert enemy here -->
+      </div> <!-- end of box-->
+
+      <p>insert instructions for game</p>
+
+    </div> <!-- end of col-->
+  </div> <!-- end of row-->
+</div> <!-- end of container-fluid -->
+
+
+
+  <script src="js/game.js"></script>
+  <script src="js/app.js"></script>
```

Comment:
@tonianner you have a file called `more_functions.js` but you don't include the file from the HTML, why?